<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSearch</class>
 <widget class="QWidget" name="UserSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="2">
    <widget class="QPushButton" name="flightTimeSort">
     <property name="text">
      <string>起飞时间排序</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="userSearchReturn">
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLineEdit" name="endPlace"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>出发地</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="id"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="startPlace"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="company"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>到达地</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>公司</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>航班号</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>舱位</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="seatType">
       <item>
        <property name="text">
         <string>经济舱</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>商务舱</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>头等舱</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>机型</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="type"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>起飞日期</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="startTime"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>乘机人姓名</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="buyer"/>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="userSearch">
     <property name="text">
      <string>发起查询</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="userInfo">
     <property name="text">
      <string>个人信息</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="userSearchBuy">
     <property name="text">
      <string>我的订单</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="add">
     <property name="text">
      <string>将该航班加入订单</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="flightPriceSort">
     <property name="text">
      <string>最低价格排序</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
